google/cloud/bigtable/admin/integration_tests/admin_backup_integration_test.cc

```diff
@@ -129,8 +132,12 @@ TEST_F(AdminBackupIntegrationTest, CreateListGetUpdateRestoreDeleteBackup) {
   backup = client_.GetBackup(backup_name);
   ASSERT_STATUS_OK(backup);
   EXPECT_EQ(backup->name(), backup_name);
-  EXPECT_THAT(backup->expire_time(),
-              IsProtoEqual(ToProtoTimestamp(expire_time)));
+  // The proto documentation says backup expiration times are in "microseconds
+  // granularity":
+  //   https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#google.bigtable.admin.v2.Backup
+  auto const delta = expire_time - ToChronoTimePoint(backup->expire_time());
+  EXPECT_THAT(delta, AllOf(Le(std::chrono::microseconds(1)),
+                           Ge(-std::chrono::microseconds(1))));
```

Comment:
An alternative to consider:

If we start off with ...
```
  auto expire_time = std::chrono::time_point_cast<std::chrono::milliseconds>(
      std::chrono::system_clock::now() + std::chrono::hours(12));
```
then I think everything else will fall out the way that it was.

Reply:
s/milliseconds/microseconds/